pyfolio/perf_attrib.py

```diff
@@ -92,10 +108,13 @@ def perf_attrib(returns, positions, factor_returns, factor_loadings):
             2017-01-01  0.249087  0.935925        1.185012          1.185012
             2017-01-02 -0.003194 -0.400786       -0.403980         -0.403980
     """
-    # convert holdings to percentages, and convert positions to long format
-    positions = get_percent_alloc(positions)
+    if pos_in_dollars:
+        # convert holdings to percentages, and convert positions to long format
```

Comment:
What does 'convert positions to long format' mean here?

Reply:
that part of the comment applies to the line `positions = positions.stack()`, it means change it from wide format:
```
                AAPL       TLT       XOM
2017-01-01  0.333333  0.568627  0.098039
2017-01-02  0.188034  0.658120  0.153846
2017-01-03 -0.263158  0.473684  0.526316
```

to long format (everything in one column with a multi index)
```
2017-01-01  AAPL    0.333333
            TLT     0.568627
            XOM     0.098039
2017-01-02  AAPL    0.188034
            TLT     0.658120
            XOM     0.153846
2017-01-03  AAPL   -0.263158
            TLT     0.473684
            XOM     0.526316
```